pandas/_libs/lib.pyx

```diff
@@ -107,18 +109,36 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 def is_scalar(val: object) -> bint:
     """
     Return True if given value is scalar.
+    
+    Parameters
+    ----------
+    val : numpy array scalar (e.g. np.int64), Python builtin numerics,
```

Comment:
@datapythonista @jorisvandenbossche is this how we format lists of things?

Reply:
@brute4s99 this should be like:

```
val : single line type description
    The full parameter description with indent of 4 spaces (this can
    span over multiple lines)
```

If the type description is a long enumeration (like in this case) and doesn't fit on a single line, you can keep the actual type description vague (eg "scalar"), and then put all the possibilities in the parameter description.

Reply:
thanks for clarification, @jorisvandenbossche ! Will follow suit .